### PR TITLE
Add Yamale

### PR DESCRIPTION
**Pull request by adament**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/yamale/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ac7dbb7b01c9fc61a5954d61fc6deb66b177c1f84146925add92dc0fa2b7e4c3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt"
```
Comment (npavlovikj) [inline recipes/yamale/meta.yaml]: Please use `script: "{{ PYTHON }} -m pip install . -vv"` instead, https://conda-forge.readthedocs.io/en/latest/meta.html#use-pip.

File: recipes/yamale/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  commands:
+    - yamale -h
+
+about:
+  home: https://github.com/23andMe/Yamale
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'A schema and validator for YAML.'
+
```
Comment (npavlovikj) [inline recipes/yamale/meta.yaml]: Remove empty line.

**Comment by npavlovikj** [comment]
@adament , I left minor comments - we can merge the recipe afterwards.

**Comment by adament** [comment]
@npavlovikj Thank you very much your review, I have attempted to incorporate your requested changes.

**Comment by npavlovikj** [comment]
The recipe LGTM @adament , thank you for contributing 🎉

**Pull request merged**